.github/workflows/rm-prime_2.8_and_2.9.yaml

```diff
@@ -0,0 +1,59 @@
+# This workflow calls the master E2E workflow with custom variables
+name: UI-RM_PRIME_2.8_AND_2.9
+run-name: ${{ github.event_name == 'workflow_dispatch' && format('`{0}` on `{1}` - `{2}` destroy={3}', inputs.rancher_version, inputs.upstream_cluster_version, inputs.grep_test_by_tag, inputs.destroy_runner) || github.workflow }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      qase_run_id:
+        description: Qase run ID where the results will be reported. (Please update with custom run ID.)
+        default: none
+        type: string
+      destroy_runner:
+        description: Destroy the auto-generated self-hosted runner
+        default: true
+        type: boolean
+      rancher_version:
+        description: Rancher version channel/version/head_version latest/2.9.x[-rc1], prime/2.9.x, prime/devel/2.9, alpha/2.9.0-alpha1
+        default: prime/2.9.1
+        type: string
+        required: true
+      upstream_cluster_version:
+        description: K3s upstream cluster version where to install Rancher
+        default: 'v1.30.3+k3s1'
```

Comment:
do we want also `1.28` to be 2.8 compatible or?

Reply:
Yes, I will consider K8s 1.28 version support for both versions 2.8 and 2.9.

Reply:
Updated it.